airbrake/utils.py

```diff
@@ -16,6 +17,16 @@
     TypeType = type
 
 
+class FailProofJSONEncoder(json.JSONEncoder):
+    """Uses object's representation for unsupported types."""
+
+    def default(self, o):  # pylint: disable=E0202
+        # E0202 ignored in favor of compliance with documentation:
+        # https://docs.python.org/2/library/json.html#json.JSONEncoder.default
+        """Return object's repr when not JSON serializable."""
+        return repr(o)
```

Comment:
I agree, I have a question though:

If I do `except Exception:` or just `except:`, Pylint will complain about it with `Catching too general exception Exception` or `No exception type(s) specified`, which is reasonable. But at this point `o` may be any object and we have no idea what type of exceptions `o.__repr__()` may raise.

Is it OK if just add an inline comment disabling the linter check?